.github/workflows/release.yml

```diff
@@ -44,11 +49,11 @@ jobs:
           echo "using version tag ${GITHUB_REF:10}"
           echo ::set-output name=version::"${GITHUB_REF:10}"
 
-      - name: Install Rust
-        if: matrix.rust
+      - name: Update rust
         run: |
           rustup update ${{ matrix.rust }} --no-self-update
           rustup default ${{ matrix.rust }}
+          rustup target add wasm32-unknown-unknown --toolchain stable
```

Comment:
This shouldn't be needed since you're using the `actions-rs/toolchain` above, right?

Reply:
Correct! Nice catch. Thanks!